renderer/reducers/lnd.js

```diff
@@ -168,7 +166,7 @@ export const connectGrpcService = lndConfig => async dispatch => {
   } catch (error) {
     dispatch({ type: CONNECT_GRPC_FAILURE, error })
     // Disconnect event listeners.
-    unsubFromGrpcEvents(grpc)
+    // unsubFromGrpcEvents(grpc)
```

Comment:
is that commented for purpose?

Reply:
actually no. that's merge issue, so good catch thanks!